src/time-picker/TimePicker.js

```diff
@@ -155,6 +156,8 @@ export const filterTime = ({ originalInput, minTime, maxTime }) => time => {
   return lteTime(minTime, time) && lteTime(time, maxTime) && startsWith(time, originalInput);
 };
 
+export const sortTimeList = timeList => sortBy(timeList, time => ( time.hours === 0 ? 24 : time.hours));
```

Comment:
> aren't you breaking the order of H24 options this way?
No, because H24 goes from 0 to 23.
> Also, why are you exporting it?
Yes is because i want to improve tests, ill do here